crypto-test/.jvm/src/test/scala/org/bitcoins/crypto/JvmECPublicKeyTest.scala

```diff
@@ -1,10 +1,7 @@
 package org.bitcoins.crypto
 
-import org.bitcoins.testkitcore.gen.CryptoGenerators
-import org.bitcoins.testkitcore.util.BitcoinSUnitTest
-
 /** Public key tests specific to the JVM */
-class JvmECPublicKeyTest extends BitcoinSUnitTest {
+class JvmECPublicKeyTest extends BitcoinSCryptoTest {
```

Comment:
nit: On line 16, we should probably use `fail()` if we have an unexpected crypto runtime rather than `succeed`?